eng/common/tools.ps1

```diff
@@ -192,8 +192,6 @@ function InitializeDotNetCli([bool]$install, [bool]$createSdkLocationFile) {
     if (-not (Test-Path(Join-Path $dotnetRoot "sdk\$dotnetSdkVersion"))) {
       if ($install) {
         InstallDotNetSdk $dotnetRoot $dotnetSdkVersion
-        # Installing SDK 5 to cover the build of all projects within the repo 
-        InstallDotNetSdk $dotnetRoot "5.0.408"
```

Comment:
It looks like installing .NET SDK 5 by the associated change  in https://github.com/dotnet/msbuild/pull/11658 is a temporary workaround to make build pass. This can be resolved by updating the runtime to 5.0 in global.json.